examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/ContentApp.java

```diff
@@ -0,0 +1,13 @@
+/** a.k.a content app like the PV app on a TV */
+public class ContentApp {
```

Comment:
For reference : https://github.com/project-chip/connectedhomeip/blob/master/examples/tv-app/android/App/platform-app/src/main/java/com/matter/tv/server/model/ContentApp.java
Looks like all the properties are present. Maybe have a Set for cluster IDs to make clear that they cant be duplicate.

Reply:
I am actually using the other option we discussed: that of letting the PV app fetch the attributes from the ApplicationBasic cluster directly, rather than packaging them in to this one class. That seems to be a simpler option, and the PV app can decide what information it needs to fetch. This ContentApp class will just contain the endpointId and the list of clusterIds.